package io.defitrack.exception

class ExceptionResult(val error: String)